test/distributed/fsdp/test_fsdp.py

```diff
@@ -0,0 +1,259 @@
+import functools
+
+import sys
+import torch
+import torch.nn as nn
+
+import torch.distributed as dist
+from torch.distributed._fsdp import FullyShardedDataParallel
+from torch.distributed._fsdp.fully_sharded_data_parallel import TrainingState
```

Comment:
should the TrainingState be exposed on the main fsdp module if you need it here?